src/view/movies/CollectionsView.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, useCallback } from 'react';
+
+import ViewItemsContainer from '../components/ViewItemsContainer';
```

Comment:
I think these should go in `src/routes/movies`... I know they don't currently represent routes really, but imho it would make sense for them to be someday.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3792/commits/d7e48d30b6480d3af9083b3aa36eebb4331a2d3f